### PR TITLE
Django and uWSGI with Nginx on Ubuntu 14.04

### PR DESCRIPTION
**Pull request by thetechstewart**
This is an import from a submission given to us via email. It has not yet been reviewed for technical accuracy or edited for for proper formatting. It's purpose here is to gauge interest from the community on this subject, and solicit independent review of the contents.

### CODE REVIEW

**Comment by spariev** [comment]
Hi, author there, any comments and suggestions on improving this guide are welcomed

**Comment by cwlinode** [comment]
@spariev Thank you for your submission. Your guide has been added to our queue for review within our documentation team.

**Comment by alexfornuto** [comment]
@cwlinode has emailed the author about a technical issue.

**Comment by spariev** [comment]
@alexfornuto @cwlinode I indeed received an email but it was related to my another pull request (namely https://github.com/linode/docs/pull/328 ). You may want to update the labels accordingly.

**Comment by cwlinode** [comment]
@spariev We've run into an issue.

The last action is to access the server from a web browser over port 80 but this just serves the nginx welcome page. At that point, the Django "It works." page is also not accessible on port 8080 unless  './manage.py runserver..' is rerun, but the page was working at the end of the *Set up Sample Django Application* step.

File: docs/websites/nginx/django-uwsgi-nginx-ubuntu-14.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        location / {
+			include         uwsgi_params;
+			uwsgi_pass      unix:/home/django/sample/sample.sock;
+		}
+	}
+	~~~
+
+2.  Link newly created configuration file to `sites-enabled` directory to enable it:
+
+		sudo ln -s /etc/nginx/sites-available/sample /etc/nginx/sites-enabled
```
Comment (spariev) [inline docs/websites/nginx/django-uwsgi-nginx-ubuntu-14.md]: Also at this point default nginx site config should be removed with `sudo rm /etc/nginx/sites-enabled/default`

File: docs/websites/nginx/django-uwsgi-nginx-ubuntu-14.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+{: .file}
+/etc/nginx/sites-available/sample
+:   ~~~ conf
+	server {
+		listen 80;
+		server_name example.com;
+
+	    location = /favicon.ico { access_log off; log_not_found off; }
+		location /static/ {
+			root /home/user/sample;
```
Comment (spariev) [inline docs/websites/nginx/django-uwsgi-nginx-ubuntu-14.md]: `/home/user/sample` should be changed to `/home/django/sample` here

**Comment by spariev** [comment]
With changes above (check the path to root in nginx site config and remove default nginx site) everything should work. If it still does not, please take a look at the uWSGI logs at `/var/log/uwsgi.log` and post here any error messages you'll see.

**Comment by cwlinode** [comment]
@sparlev Thank you, everything is working great. This guide is now under copy edit review.

**Comment by thetechstewart** [comment]
@spariev - I've pulled in the final draft from @cwlinode for you to review.  We can publish the final draft of the guide as soon as you give up the go-ahead.

**Comment by spariev** [comment]
@thetechstewart everything looks fine for me, thanks for correcting my typos and grammar mistakes.

**Pull request merged**